src/hooks/useListenItems.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+//firebase
+import { db } from '../lib/firebase';
+import { onSnapshot, collection, query, where } from 'firebase/firestore';
+import useToken from './useToken';
+
+const useListenItems = () => {
```

Comment:
It took me awhile to figure out what useListenItems meant, could we call this useListenToItems or useFetchItems?

Reply:
useFetchItems is the clearest option in my opinion. Thank you for your suggestion!

Reply:
I agree with `useFetchItems`